src/modules/purge.js

```diff
@@ -39,5 +39,26 @@ module.exports = bot => {
 	}
   });
 
-  bot.registerCommandAlias('p', 'purge');
+	bot.registerCommandAlias('p', 'purge');
+	
+	addInboxServerCommand('undo', async (msg, args, thread) => {
```

Comment:
isn't this just the purge command but not admin locked?

Reply:
Just 1 message

Reply:
See L53

Reply:
yes but you could just do it in quick succession

Reply:
i also apparently can’t read, i shouldn’t be doing these while trying to note down stellar classifications - my oopsie